docs/src/guides/run-analysis-on-terra.rst

```diff
@@ -64,8 +70,36 @@ Connect your data files to the WDL workflow
   |Nextstrain_WRKFLW|  sequence_fasta  | File  | this.sequences       |
   +-----------------+------------------+-------+----------------------+
 
-10. Click on the **OUTPUTS** tab
-11. Connect your generated output back to the data table, but filling in values:
+10. If creating a build with multiple sequence and metadata files, can upload a targz folder containing the files. Otherwise skip
```

Comment:
This seems like a really cool feature! What if you included an example here of what the tar archive's structure should be, so people have a sense of how to prepare this file prior to upload? The expression "targz folder" makes me think that I would need to place my files in a directory and then tar and gzip that directory. Is that correct?

We should also figure out how we want to consistently refer to compressed tar archives throughout the docs. I know what you mean by "targz folder" but we sometimes use "tarball" or "tar archive", etc.

Reply:
Thank you for the feedback! And yes, that is correct!

I'm thinking through how to add an example of the tarball structure. Originally, [the tarball contextual sequences feature was added](https://github.com/nextstrain/ncov/pull/951) by [this request](https://bedfordlab.slack.com/archives/C01G4B4UY31/p1653350349901749).

I'll probably add it as a part 2... maybe something like:

> Now that you've worked through the minimal case, here is an example of adding contextual sequences and a configfile in a tarball. The structure of the directory must be ...

I'll think through it a bit more

Reply:
Could we move:

> Now that you've worked through the minimal case, here is an example of adding contextual sequences and a configfile in a tarball. The structure of the directory must be ...

As something to-be-done in a future PR?